uat/t_utils.py

```diff
@@ -0,0 +1,47 @@
+from pathlib import Path
+
+import boto3
+
+
+def clean_up_aws_resources(component_name, component_version, region):
+    sts_client = boto3.client("sts", region_name=region)
+    caller_identity_response = sts_client.get_caller_identity()
+    account_num = caller_identity_response["Account"]
+    delete_component(component_name, component_version, region, account_num)
+    delete_s3_artifact(region, account_num, component_name, component_version)
+
+
+def delete_s3_artifact(region, account, component_name, component_version):
+    s3_client = boto3.client("s3", region_name=region)
+    try:
+        bucket = f"gdk-cli-uat-{region}-{account}"
+        res = s3_client.list_objects(Bucket=bucket)
+        if "Contents" not in res:
+            return
+        for f in res["Contents"]:
+            if f"{component_name}/{component_version}" in f["Key"]:
+                s3_client.delete_object(Bucket=bucket, Key=f["Key"])
+    except Exception as e:
```

Comment:
Will this fail the run or report it as passed? Don't want this to go under the radar if it fails to clean up

Reply:
Yes, right now the test fails when the resources are not properly cleaned up.